### PR TITLE
Unified release 96

### PR DESCRIPTION
**Pull request by thomasmaurel**

### CODE REVIEW

File: misc_scripts/get_list_databases_for_division.pl [hunk 1]
```diff
@@ -129,44 +129,20 @@ =head1 VERSION
 }
 
 #Get the release for the given division
-if ( $division eq "vertebrates") {
-  if (defined $opts->{release}){
+#Get the release for the given division
```
Comment (marcoooo) [inline misc_scripts/get_list_databases_for_division.pl]: Duplicated comment ?

File: misc_scripts/get_list_databases_for_division.pl [hunk 2]
```diff
@@ -129,44 +129,20 @@ =head1 VERSION
...
+if (defined $opts->{release}){
+  $release_info = $rdba->fetch_by_ensembl_genomes_release($opts->{release});
```
Comment (marcoooo) [inline misc_scripts/get_list_databases_for_division.pl]: A chance that we don't have a release 42 for e! :-)

**Comment by marcoooo** [comment]
A general note, It seems that code a very similar in terms of done tests. In the future, we should make this more centralized.

File: misc_scripts/get_list_databases_for_division.pl (hunk 1 above)
Comment (thomasmaurel) [reply misc_scripts/get_list_databases_for_division.pl]: oops, bad copy-paste I guess

File: misc_scripts/get_list_databases_for_division.pl (hunk 2 above)
Comment (thomasmaurel) [reply misc_scripts/get_list_databases_for_division.pl]: No, it's a feature. If you don't give a release number, the metadata database will return the current release.

**Pull request merged**